MapboxNavigation/RouteMapViewController.swift

```diff
@@ -79,34 +78,34 @@ class RouteMapViewController: UIViewController {
         wayNameView.layer.borderColor = UIColor.lightGray.cgColor
         wayNameView.applyDefaultCornerRadiusShadow()
         wayNameLabel.layer.masksToBounds = true
-    }
-
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
         
-        mapView.compassView.isHidden = true
-        mapView.addAnnotation(destination)
-
+        
+        mapView.setContentInset(contentInsets, animated: false)
+        mapView.setUserLocationVerticalAlignment(.bottom, animated: false)
+        mapView.setUserTrackingMode(.followWithCourse, animated: false)
+        
         if let camera = pendingCamera {
             mapView.camera = camera
         } else {
-            let camera = tiltedCamera
-            if let coordinates = route.coordinates, coordinates.count > 1 {
-                camera.centerCoordinate = coordinates.first!
-                camera.heading = coordinates[0].direction(to: coordinates[1])
+            let camera = MGLMapCamera()
+            if let userLocation = mapView.userLocation {
+                camera.centerCoordinate = userLocation.coordinate
+                if let course = userLocation.location?.course {
+                    camera.heading = course
+                }
             }
+            camera.pitch = 60
+            camera.altitude = 600
             mapView.setCamera(camera, animated: false)
         }
+        mapView.manuallyUpdatesLocation = true
```

Comment:
Order matters here. This needs to be called after `mapView.setUserTrackingMode(.followWithCourse, animated: false)`